File: Script/Fibonacci_ricorsivo/Fibonacci.py
import sys

def fibonacci(n):
    if n <= 1:
        return n
    return fibonacci(n-1) + fibonacci(n-2)

if __name__ == "__main__":
    if len(sys.argv) != 2:
        print(f"Uso: python fibonacci_rec.py <numero>")
        sys.exit(1)
    n = int(sys.argv[1])
    print(f"Fibonacci({n}) = {fibonacci(n)}")

File: Script/Fibonacci_ricorsivo/Fibonacci.c
#include <stdio.h>
#include <stdlib.h>

int fibonacci(int n) {
    if (n <= 1) return n;
    return fibonacci(n-1) + fibonacci(n-2);
}

int main(int argc, char *argv[]) {
    if (argc != 2) {
        printf("Uso: %s <numero>\n", argv[0]);
        return 1;
    }
    int n = atoi(argv[1]);
    printf("Fibonacci(%d) = %d\n", n, fibonacci(n));
    return 0;
}
